src/OpenTelemetry/BackwardCompatibilitySwitches.cs

```diff
@@ -0,0 +1,28 @@
+// <copyright file="BackwardCompatibilitySwitches.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry
+{
+    public static class BackwardCompatibilitySwitches
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether activity status switch is enabled
+        /// If true then activity Status and StatusDescription properties will be set
+        /// using tags otel.status_code and otel.status_description respectively.
+        /// </summary>
+        public static bool ActivityStatusSwitch { get; set; } = true;
```

Comment:
Trying to think of a more descriptive name. How about "StatusTagMigrationEnabled"?

I think for the comments something like...

```csharp
        /// <summary>
        /// Gets or sets a value indicating whether or not activity status migration is enabled. Default value: true.
        /// </summary>
        /// <remarks>
        /// If true then <see cref="Activity.Status"/> and <see cref="Activity.StatusDescription"/> properties (added in .NET 6) will be set
        /// from `otel.status_code` and `otel.status_description` tag values respectively prior to export. For more details see <a href="migration_doc_url_here">migration doc name here</a>.
        /// </remarks>
```

Reply:
Looks good - let me know if you have a suggestion for class name as well :)